atlasdb-config/src/main/java/com/palantir/atlasdb/factory/TransactionManagers.java

```diff
@@ -106,7 +108,7 @@ public static SerializableTransactionManager create(
             Set<Schema> schemas,
             Environment env,
             boolean allowHiddenTableAccess) {
-        DebugLogger.logger.info("Called TransactionManagers.create on thread {}. This should only happen once.",
+        log.info("Called TransactionManagers.create on thread {}. This should only happen once.",
```

Comment:
Ideally this would go to the DebugLogger, but I guess that sending it to the (probably suppressed) stdlog.info is a lesser evil than introducing the circular dependency.